swagger.yml

```diff
@@ -8,7 +8,7 @@ info:
   description: |
     Federated PDS API which provides actionable end points standardized
     between the different nodes.
-  version: 0.3.dev
+  version: keywords
```

Comment:
@tloubrieu-jpl @tdddblog shouldn't this come from swaggerhub?

Reply:
As discussed with @tloubrieu-jpl, I used "SYNC" button in SwggerHub to create this branch (SwggerHub created it). I'm not sure how `swagger.yml` is usually merged.